core/templates/dev/head/pages/base.html

```diff
@@ -82,7 +82,6 @@
         csrf_token: JSON.parse('{{csrf_token|js_string}}'),
         status_code: JSON.parse('{{status_code}}'),
         GCS_RESOURCE_BUCKET_NAME: JSON.parse('{{GCS_RESOURCE_BUCKET_NAME|js_string}}'),
-        iframed: JSON.parse('{{iframed|js_string}}'),
```

Comment:
Changes made in base.html should be replicated [here](https://github.com/oppia/oppia/blob/develop/core/tests/build_sources/templates/base.html).

Reply:
Okay, thanks for noting that. I'll make the change there too.